### ISSUE TITLE
Configuration for angularjs projects

### ISSUE THREAD

**Issue by francittadini** [hbenl/vscode-jasmine-test-adapter#18]
I cannot get the proper configuration for my angularjs project. How do I import angular-mocks? Also, how do I tell the extension where are my sources?

Thanks!

**Comment by hbenl** [comment]
What do you do to run your tests on the command line?

**Comment by francittadini** [comment]
I'm using Chutzpah Test Runner in Visual Studio. I don't know if you are familiar with it, but you have one config where you setup your external libs (like angular-mocks or jasmine) and your source files. I'm trying to make everything work in VSCode.

**Comment by francittadini** [comment]
Yeah, that sounds logic. Thanks for your answers 😃

**Comment by SylChamber** [comment]
@francittadini

Actually, it is possible to run AngularJS in Node.js for more developer friendly testing, by using jsdom to simulate the browser. That's the technique being used for testing by the React.js and (it seems) the Vue.js communities.

However, test setup for AngularJS unit testing in Node is difficult, and references are difficult to find.

This summer, I finally succeeded in running tests using Jest and Cucumber.js, but I had difficulties with the Jasmine test adapter for VS Code. It was working when in debug, but not when running the tests. 

Since I really like the Jest extension's "live testing" mode, I did not pursue my efforts with the Jasmine test explorer. But I would really like to make it work, because I really like it's test explorer.

I've been meaning to post an article on testing AngularJS with jsdom. Ping me after the weekend. Same handle on Twitter as on GitHub.
